install/kubernetes/cilium/values.yaml

```
@@ -2147,6 +2147,20 @@ clustermesh:
       useDigest: false
       pullPolicy: "Always"
 
+    # -- Specifies the security context of the api server pod
+    securityContext:
+      runAsUser: 1000
+      runAsGroup: 1000
+      fsGroup: 1000
```

Once switched to distroless, these should be 65532.
```suggestion
      runAsUser: 65532
      runAsGroup: 65532
      fsGroup: 65532
```